src/components/link-icon/index.js

```diff
@@ -7,7 +7,7 @@ import './index.css';
 const LinkIcon = ({ to, newTab, icon = <></>, className = '', ...props }) => (
   <Link
     className={'link_icon ' + className}
-    target={newTab ? '_blank' : undefined}
+    // target={newTab ? '_blank' : undefined}
```

Comment:
This line will be used later, I suppose?

Reply:
Eventually I'd like the model/gene/etc info buttons to always open in a new tab so they don't disrupt normal navigation of the app. But when testing PRs on Netlify, I don't have the redirects set up for all the urls, so I want to allow the user to stay in the same tab the whole time.

I'll uncomment this much later when we get close to releasing. (And eventually I guess I'll try to figure out how to set up redirects in Netlify).